MergerLogicUnitTests/Utils/GpkgUtilsTest.cs

```diff
@@ -232,7 +232,7 @@ public void GetLastTile(Coord baseCoords)
         public void GetTileCount(int tileCount)
         {
             string path = this.GetGpkgPath();
-            var testTiles = new List<Tile>(tileCount);
+            var testTiles = new List<Tile>(tileCount); // todo: discuss how to check for long
```

Comment:
irrelevant for this line.
in general the only reasonable thing that is related to long here that can be tested is that empty list is return from get batch when long offset is specified